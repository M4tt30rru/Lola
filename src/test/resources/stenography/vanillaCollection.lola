##Import "std.lola"
##Find ##Any(_mods) ##CollectionType(_col)<##Any(_gen)> ##Identifier(_name) = empty;
 ##replace ##(_mods) ##(_col)<##(_gen)> ##(_name) = new ##('HashSet' if _col.name.strip() == 'Set' else 'HashMap' if _col.name.strip() == 'Map' else 'LinkedList')<>();

private transient Map<X, Y> m = empty;
