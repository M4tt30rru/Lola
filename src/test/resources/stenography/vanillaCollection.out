private transient Map<X, Y> m = new HashMap<>();
